### PR TITLE
Fixes #572 - Allow appending rows after a dynamic column was inserted

### LINKED ISSUE #572: appending rows when there are dynamic columns
At the moment we cannot add rows (using append, rpush, lpush) when there is already a dynamic column: a function object can be added at the appropriate place in the row, but unlike a normal dynamic column it will not be executed.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: src/tablib/core.py [hunk 1]
```diff
@@ -547,6 +553,7 @@ def insert_col(self, index, col=None, header=None):
 
         # Callable Columns...
         if hasattr(col, '__call__'):
+            self._dyn_columns[self.width] = col
```
Comment (hugovk) [inline src/tablib/core.py:556]:
```suggestion
            self._dynamic_columns[self.width] = col
```

**Review by hugovk — APPROVED** [review]
Looks good, couple of small suggestions.

**Comment by claudep** [comment]
Thanks for the review, but I'd like to work on it some more, specifically to allow the use case of people inserting complete new rows with dynamic values already calculated, and add some docs about it.

**Comment by claudep** [comment]
A second quick review would be great (also for wording).

**Review by hugovk — APPROVED** [review]
Wording is fine 👍

**Pull request merged**